<launch>
    <param name="lat_zero" value="0.0"/>
    <param name="lon_zero" value="0.0"/>
    <param name="alt_zero" value="0.0"/>

    <node pkg="rviz" name="rviz" type="rviz" args="-d $(find first_project)/rviz_config.rviz"/>

    <node pkg="first_project" name="gps_to_odom" type="gps_to_odom" output="screen"/>

    <node pkg="tf" type="static_transform_publisher" name="odom_to_tf" args="0 0 0 0 0 0 /map /odom 100" />
</launch>
